src/test/renderWithProviders.tsx

```diff
@@ -179,27 +184,35 @@ function DefaultProviders({ children, store, queries, queryClient, Router = Defa
       <ReduxProvider store={store}>
         <LanguageProvider>
           <MuiThemeProvider theme={theme}>
-            <ExprContextWrapper>
-              <ApplicationMetadataProvider>
-                <OrgsProvider>
-                  <ApplicationSettingsProvider>
-                    <LayoutSetsProvider>
-                      <ProfileProvider>
-                        <PartyProvider>
-                          <TextResourcesProvider>
-                            <FooterLayoutProvider>
-                              <Router>
-                                <InstantiationProvider>{children}</InstantiationProvider>
-                              </Router>
-                            </FooterLayoutProvider>
-                          </TextResourcesProvider>
-                        </PartyProvider>
-                      </ProfileProvider>
-                    </LayoutSetsProvider>
-                  </ApplicationSettingsProvider>
-                </OrgsProvider>
-              </ApplicationMetadataProvider>
-            </ExprContextWrapper>
+            <PageNavigationProvider>
+              <Router>
+                <ExprContextWrapper>
+                  <ApplicationMetadataProvider>
+                    <OrgsProvider>
+                      <ApplicationSettingsProvider>
+                        <LayoutSetsProvider>
+                          <ProfileProvider>
+                            <PartyProvider>
+                              <TextResourcesProvider>
+                                <FooterLayoutProvider>
+                                  <LayoutsProvider>
+                                    <LayoutSettingsProvider>
```

Comment:
Wouldn't this mean that the renderWithoutLayouts* function now renders with layouts?

Reply:
Yes it would! I've changed it to render `LayoutsProvider`, `LayoutSettingsProvider` and `UiConfigProvider` to `renderWithInstanceAndLayout` :smile: